<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="speechNameLabel">
       <property name="text">
        <string>Speech Name : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="speechNameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="authorEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="promptGroupBox">
     <property name="title">
      <string>Prompts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="promptTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QGroupBox" name="speechNotesGroupBox">
     <property name="title">
      <string>Speech Notes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextEdit" name="speechNotesTextEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="wordCountLabel">
        <property name="text">
         <string>Word count:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
